<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="10" ignored="0" total="10" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="303" started-at="2020-04-26T17:38:31 PDT" finished-at="2020-04-26T17:38:31 PDT">
    <groups>
    </groups>
    <test name="Default test" duration-ms="303" started-at="2020-04-26T17:38:31 PDT" finished-at="2020-04-26T17:38:31 PDT">
      <class name="core.PrimeNumberCheckerTest">
        <test-method status="FAIL" signature="test(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:core.PrimeNumberCheckerTest@1a1da881]" name="test" duration-ms="79" started-at="2020-04-26T17:38:31 PDT" data-provider="Prime Numbers" finished-at="2020-04-26T17:38:31 PDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[TC-01.01 (2 is prime number)]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[2]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[true]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: "./primes_10.csv"
     but: was "abcd"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: "./primes_10.csv"
     but: was "abcd"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at core.PrimeNumberCheckerTest.test(PrimeNumberCheckerTest.java:35)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:71)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:13)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
        <test-method status="FAIL" signature="test(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:core.PrimeNumberCheckerTest@1a1da881]" name="test" duration-ms="2" started-at="2020-04-26T17:38:31 PDT" data-provider="Prime Numbers" finished-at="2020-04-26T17:38:31 PDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[TC-01.02 (3 is prime number)]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[true]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: "./primes_10.csv"
     but: was "abcd"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: "./primes_10.csv"
     but: was "abcd"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at core.PrimeNumberCheckerTest.test(PrimeNumberCheckerTest.java:35)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:71)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:13)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
        <test-method status="FAIL" signature="test(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:core.PrimeNumberCheckerTest@1a1da881]" name="test" duration-ms="3" started-at="2020-04-26T17:38:31 PDT" data-provider="Prime Numbers" finished-at="2020-04-26T17:38:31 PDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[TC-01.03 (4 is not prime number)]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[4]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[false]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: "./primes_10.csv"
     but: was "abcd"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: "./primes_10.csv"
     but: was "abcd"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at core.PrimeNumberCheckerTest.test(PrimeNumberCheckerTest.java:35)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:71)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:13)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
        <test-method status="FAIL" signature="test(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:core.PrimeNumberCheckerTest@1a1da881]" name="test" duration-ms="3" started-at="2020-04-26T17:38:31 PDT" data-provider="Prime Numbers" finished-at="2020-04-26T17:38:31 PDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[TC-01.04 (5 is prime number)]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[5]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[true]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: "./primes_10.csv"
     but: was "abcd"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: "./primes_10.csv"
     but: was "abcd"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at core.PrimeNumberCheckerTest.test(PrimeNumberCheckerTest.java:35)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:71)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:13)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
        <test-method status="FAIL" signature="test(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:core.PrimeNumberCheckerTest@1a1da881]" name="test" duration-ms="3" started-at="2020-04-26T17:38:31 PDT" data-provider="Prime Numbers" finished-at="2020-04-26T17:38:31 PDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[TC-01.05 (6 is not prime number)]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[6]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[false]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: "./primes_10.csv"
     but: was "abcd"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: "./primes_10.csv"
     but: was "abcd"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at core.PrimeNumberCheckerTest.test(PrimeNumberCheckerTest.java:35)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:71)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:13)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
        <test-method status="FAIL" signature="test(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:core.PrimeNumberCheckerTest@1a1da881]" name="test" duration-ms="2" started-at="2020-04-26T17:38:31 PDT" data-provider="Prime Numbers" finished-at="2020-04-26T17:38:31 PDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[TC-01.06 (7 is prime number)]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[7]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[true]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: "./primes_10.csv"
     but: was "abcd"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: "./primes_10.csv"
     but: was "abcd"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at core.PrimeNumberCheckerTest.test(PrimeNumberCheckerTest.java:35)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:71)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:13)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
        <test-method status="FAIL" signature="test(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:core.PrimeNumberCheckerTest@1a1da881]" name="test" duration-ms="4" started-at="2020-04-26T17:38:31 PDT" data-provider="Prime Numbers" finished-at="2020-04-26T17:38:31 PDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[TC-01.07 (8 is not prime number)]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[8]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[false]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: "./primes_10.csv"
     but: was "abcd"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: "./primes_10.csv"
     but: was "abcd"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at core.PrimeNumberCheckerTest.test(PrimeNumberCheckerTest.java:35)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:71)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:13)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
        <test-method status="FAIL" signature="test(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:core.PrimeNumberCheckerTest@1a1da881]" name="test" duration-ms="3" started-at="2020-04-26T17:38:31 PDT" data-provider="Prime Numbers" finished-at="2020-04-26T17:38:31 PDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[TC-01.08 (9 is not prime number)]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[9]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[false]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: "./primes_10.csv"
     but: was "abcd"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: "./primes_10.csv"
     but: was "abcd"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at core.PrimeNumberCheckerTest.test(PrimeNumberCheckerTest.java:35)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:71)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:13)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
        <test-method status="FAIL" signature="test(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:core.PrimeNumberCheckerTest@1a1da881]" name="test" duration-ms="3" started-at="2020-04-26T17:38:31 PDT" data-provider="Prime Numbers" finished-at="2020-04-26T17:38:31 PDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[TC-01.09 (10 is not prime number)]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[10]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[false]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: "./primes_10.csv"
     but: was "abcd"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: "./primes_10.csv"
     but: was "abcd"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at core.PrimeNumberCheckerTest.test(PrimeNumberCheckerTest.java:35)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:71)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:13)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
        <test-method status="FAIL" signature="test(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:core.PrimeNumberCheckerTest@1a1da881]" name="test" duration-ms="2" started-at="2020-04-26T17:38:31 PDT" data-provider="Prime Numbers" finished-at="2020-04-26T17:38:31 PDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[TC-01.10 (11 is prime number)]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[11]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[true]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: "./primes_10.csv"
     but: was "abcd"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: "./primes_10.csv"
     but: was "abcd"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at core.PrimeNumberCheckerTest.test(PrimeNumberCheckerTest.java:35)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:71)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:13)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
      </class> <!-- core.PrimeNumberCheckerTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
